os-datavirt/module.yaml

```diff
@@ -10,3 +10,14 @@ modules:
 execute:
 - script: configure.sh
   user: '185'
+
+packages:
+      repositories:
+          - jboss-os
+      install:
+          - hostname
```

Comment:
I think it would be preferable to move this into another module and just reference it from all the modules that use it.  It may be that everything references some common eap module and it can just go there.

Reply:
I will move it to os-eap-launch, which is included by any image which include any of os-datavirt, os-eap-launch, os-eap-sso, os-eap64-launch. os-eap7-launch (the modules using hostname, to which I had added this to).

I think we should also explicitly  make os-eap-launch a dependency of these modules themselves, to stop a theoretical future image including os-eap7-launch (say) but not os-eap-launch and breaking

Reply:
Oh of course it's already in os-eap-launch.